app/controllers/trains_controller.rb

```diff
@@ -1,15 +1,30 @@
 class TrainsController < ApplicationController
+  helper_method :train
```

Comment:
i see this here but no defined method?

Reply:
> i see this here but no defined method?

oops forgot to remove, 
i have tried to add the helper but it doesnt work even when i changed the form from `@train["name"]` to `train["name"]`

Reply:
leave it in and define it like you did in all the other controllers.
it should work if you copy how you have done it in the past